### ISSUE TITLE
Suggestions for documenting files

### ISSUE THREAD

**Issue by mikemc** [naobservatory/mgs-workflow#8]
As I read through the repo trying to understand the pipeline, I'm noticing that something that I'd find helpful is if we could have basic descriptions of more files (ideally all that are not obvious from their names/structure), either in README's or (for scripts) in comments at the start of the script.

One case where I think a README would be useful is in the 'ref/' folder. We could have 'ref/README.md' that has at least a sentence about what each of these files are. In some cases, like 'human-viruses.tsv', saying a bit more like how this file was generated and how it's used would be very handy. The README would ideally also say at the top the overall purpose of the folder / what sorts of files should go in it. (E.g. should 'libraries.csv' really be here, or should it go in 'test-local/'.

Another directory where more docs would help is scripts/. Here I think a simple 'scripts/README.md' file would work, or you could add documentation directly in the script in a comment at the top of the script.

**Comment by willbradshaw** [comment]
Yeah I agree, this is part of a general goal of more comprehensive documentation that I'm hoping to implement on the dev branch soon.

(In the specific case of `ref/human-viruses.tsv`, the dev branch version of the index workflow now generates this de novo, so this specific file will no longer exist. But I agree with the general principle.)

**Comment by willbradshaw** [comment]
Update: We now have much more thorough (but still not comprehensive) documentation, and we're expecting to write even more as part of our work on pipeline performance and usability next quarter.

**Comment by harmonbhasin** [comment]
Related to #152
